apps/model_eval_viz/app.py

```diff
@@ -152,23 +189,30 @@ def display_points(state_list:list=state_selector.value)->gv.Points:
 displayed_points = hv.DynamicMap(pn.bind(display_points, state_list=state_selector))
 
 
-def reset_map(event):
+def reset_map(event:bool)-> None:
+    '''
+    Reset the state selector when an event is triggered.
+    Args:
+        event (bool): A boolean flag to trigger the function.
+
+    Returns:
+        None.
+    '''
     if not event:
         return
     state_selector.value = []
 
 reset_button = pn.panel(pn.widgets.Button(name='Reset Map', button_type='primary'))
```

Comment:
We might have a couple other reset buttons in the future. Let's rename this variable now to reflect what exactly it is resetting.